go.mod

```diff
@@ -1,56 +1,189 @@
 module github.com/solo-io/k8s-utils
 
-go 1.14
+go 1.19
```

Comment:
do we need to go to 1.19? do we plan to bump our other repos to 1.19?

Reply:
Yeah we need 1.19 because k8s mod has a 1.19 reference :( 

This does imply we need the 1.19 bump everywhere